packages/backend/src/app/database/seeds/dev-seeds.ts

```diff
@@ -0,0 +1,32 @@
+import { ApEnvironment } from "@activepieces/shared";
+import { authenticationService } from "../../authentication/authentication.service";
+import { logger } from "../../helper/logger";
+import { system } from "../../helper/system/system";
+import { SystemProp } from "../../helper/system/system-prop";
+
+const seedDevUser = async (): Promise<void> => {
+    logger.debug("[seedDevUser]");
+
+    await authenticationService.signUp({
+        email: "dev@ap.com",
+        password: "12345678",
```

Comment:
If people skipped reading the docs, I am not sure how they would find this.

I am thinking of following options:
1- Allow sign up on development `.env` in case someone couldn't find it docs.
2- Print it in console (which might be not clear)
3- Don't merge this PR at all

Btw why this case is often for you? when running docker compose it shouldn't delete the volumes or am i wrong?

Reply:
oops! good catch!
enabled sign-up on dev `.env` in 155f3c2
logged credentials in 6554377